modules/plot/test/spec.coffee

```diff
@@ -10,6 +10,29 @@ describe "plot", ->
       array2 = [{x: 1, y1: 4, y2: 3}, {x: 2, y1: 3, y2: 4}, {x: 3, y1: 5, y2: 5}]
       array3 = [{x: 1, y: 2}, {x: 2, y: 3}, {x: 3, y: 4}, {x: 4, y: 2}]
 
+      it 'doCollisionDetection: should hide text that gets in the way of other text', ->
+        widthPx = 200
+        intendedDistance = 2
+        createSuspiciousElement = (i) ->
+          hx.detached 'div'
+            .text i
+            .style 'left', "#{i * widthPx / intendedDistance}px"
+            .style 'width', "#{widthPx}px"
+            .style 'position', 'absolute'
+            .node()
```

Comment:
new test added. I think I've made it reasonably clear why the distance is what it is.